packages/functional-tests/tests/signin/signinCached.spec.ts

```diff
@@ -15,12 +14,13 @@ test.describe('severity-2 #smoke', () => {
       syncBrowserPages: { page, settings, signin },
       testAccountTracker,
     }) => {
-      const { email } = await signInSyncAccount(
+      const credentials = await testAccountTracker.signUp();
```

Comment:
I stole these functional test changes from [this PR](https://github.com/mozilla/fxa/pull/19086/files#diff-ff4509dcb6c771c701f04fc60012dbd2a709ca0772b955a209d901db6dffc836R17) we had to revert.

We have this exact test except non-Sync in our react conversion directory, which is passing... this was going through Sync. I tested it manually in Sync and without Sync and the cached signin works as expected.